configure.ac

```diff
@@ -88,8 +96,15 @@ else
 	JANSSON_LIBS=-ljansson
 fi
 
-LIBCURL_CHECK_CONFIG(, 7.15.2, ,
-  [AC_MSG_ERROR([Missing required libcurl >= 7.15.2])])
+#LIBCURL_CHECK_CONFIG(, 7.15.2, ,
+#  [AC_MSG_ERROR([Missing required libcurl >= 7.15.2])])
+LIBCURL=-lcurl
+if test x$have_osx = xtrue
+then
+  LIBCURL_CPPFLAGS=-I/usr/local/opt/curl/include
```

Comment:
its supposed to use libcurl.pc in pkgconfig folder for the path (which doesnt look normal here)